src/feat/wave-reader.h

```diff
@@ -52,10 +52,13 @@
 
 namespace kaldi {
 
+/// For historical reasons, we scale waveforms to the range
+/// (2^15-1)*[-1, 1], not the usual default DSP range [-1, 1].
+const BaseFloat kWave0dBSampleValue = 32768.0;
```

Comment:
Could you please just call this kWaveSampleMax ?  I think it would be less confusing